session/bootstrap.go

```diff
@@ -1179,7 +1179,8 @@ func syncUpgradeState(s Session) {
 	}
 
 	for i := 0; i < retryTimes; i++ {
-		if _, err = ddl.PauseAllJobsBySystem(s); err == nil {
+		_, err = ddl.PauseAllJobsBySystem(s)
+		if err == nil || dbterror.ErrPausedDDLJob.Equal(err) {
```

Comment:
If pause 3 jobs， the first one gets `ErrPausedDDLJob`, then other jobs all don't pause.